tutorials/advanced/transforms_tutorial.py

```diff
@@ -303,12 +316,90 @@
 #
 #     trans = ax.get_xaxis_transform()
 #
+# .. _transforms-fig-scale-dpi:
+#
+# Plotting in physical units
+# ==========================
+#
+# Sometimes we want an object to be a certain physical size on the plot.
+# Here we draw the same circle as above, but in physical units.  If done
+# interactively, you can see that changing the size of the figure does
+# not change the offset of the circle from the lower-left corner,
+# does not change its size, and the circle remains a circle regardless of
+# the aspect ratio of the axes.
+
+fig, ax = plt.subplots(figsize=(5, 4))
+x, y = 10*np.random.rand(2, 1000)
+ax.plot(x, y*10., 'go', alpha=0.2)  # plot some data in data coordinates
+# add a circle in fixed-units
+circ = mpatches.Circle((2.5, 2), 1.0, transform=fig.dpi_scale_trans,
+                      facecolor='blue', alpha=0.75)
+ax.add_patch(circ)
+plt.show()
+
+###############################################################################
+# If we change the figure size...
+
+fig, ax = plt.subplots(figsize=(7, 2))
+x, y = 10*np.random.rand(2, 1000)
+ax.plot(x, y*10., 'go', alpha=0.2)  # plot some data in data coordinates
+# add a circle in fixed-units
+circ = mpatches.Circle((2.5, 2), 1.0, transform=fig.dpi_scale_trans,
+                      facecolor='blue', alpha=0.75)
+ax.add_patch(circ)
+plt.show()
+
+###############################################################################
+# Another use is putting a patch with a set physical dimmension around a
+# data point on the axes.  Here we add together two transforms.  The
+# first sets the scaling of how large the ellipse should be and the second
+# sets its position.  The ellipse is then placed at the origin, and then
+# we use the helper transform :class:`~matplotlib.transforms.ScaledTranslation`
+# to move it
+# to the right place in the ``ax.transData`` coordinate system.
+# This helper is instantiated with::
+#
+#   trans = ScaledTranslation(xt, yt, scale_trans)
+#
+# where `xt` and `yt` are the translation offsets, and `scale_trans` is
+# a transformation which scales `xt` and `yt` at transformation time
+# before applying the offsets.
+#
+# Note that in interactive use, the ellipse stays the same size even if the
+# axes limits are changed via zoom, but the ellipse's size doesn't change.
+#
+# Also note that the order of transformation matters.  Here the ellipse
+# is given the right dimensions in display space *first* and then moved
+# in data space to the correct spot.
+# If we had done the ``ScaledTranslation`` first, then
+# ``xdata[0]`` and ``ydata[0]`` would
+# first be transfromed to ``display`` coordinates (``[ 358.4  475.2]`` on
+# a 200-dpi monitor) and then those coordinates
+# would be scaled by ``fig.dpi_scale_trans`` pushing the center of
+# the ellipse well off the screen (i.e. ``[ 71680.  95040.]``).
```

Comment:
Mhh, why do you explain all the things that *would*  go wrong if you *did* something other than you're actually doing? Instead why not simply explain what you are doing and giving the coordinates for the actual case?  
I would also think this is better done *after* the example code.